requirements.yml

```diff
@@ -1,6 +1,7 @@
 ---
 collections:
   - ansible.posix
+  - ansible.utils
```

Comment:
Does this require a psa to alert the community to reinstall ansible modules?

Reply:
Yes probably :)

Reply:
I still wish we had done my idea for pre-run messages would make these things easer.